src/components/network-modifications/voltageLevel/modification/VoltageLevelModificationForm.tsx

```
@@ -5,7 +5,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+import { useState } from 'react';
 import { Grid, TextField } from '@mui/material';
```

ESLint

[error] '/home/jailuser/git/node_modules/@mui/material/index.js' imported multiple times.

(import-x/no-duplicates)